tensorflow/compiler/xla/window_util.cc

```diff
@@ -42,7 +42,7 @@ Window MakeWindow(absl::Span<const int64> sizes,
                   absl::Span<const int64> strides) {
   Window window;
   CHECK_EQ(sizes.size(), strides.size());
-  for (auto nb = 0; nb < sizes.size(); ++nb) {
+  for (auto nb = 0, auto size = sizes.size(); nb < size; ++nb) {
```

Comment:
```
tensorflow/compiler/xla/window_util.cc:45:21: error: expected unqualified-id before 'auto'
   for (auto nb = 0, auto size = sizes.size(); nb < size; ++nb) {
                     ^~~~
tensorflow/compiler/xla/window_util.cc:45:21: error: expected ';' before 'auto'
tensorflow/compiler/xla/window_util.cc:45:56: error: expected ')' before ';' token
   for (auto nb = 0, auto size = sizes.size(); nb < size; ++nb) {
                                                        ^
tensorflow/compiler/xla/window_util.cc:45:60: error: 'nb' was not declared in this scope
   for (auto nb = 0, auto size = sizes.size(); nb < size; ++nb) {
```

Let's give types here instead of `auto`